skills/declarative/calendarSkill/runtime/customaction/MSGraphComponentRegistration.cs

```diff
@@ -40,6 +40,10 @@ public class MSGraphComponentRegistration : ComponentRegistration, IComponentDec
     {
         public IEnumerable<DeclarativeType> GetDeclarativeTypes(ResourceExplorer resourceExplorer)
         {
+            yield return new DeclarativeType<CheckAvailability>(CheckAvailability.DeclarativeType);
```

Comment:
Why are these three added individually rather than based on the attribute?

Reply:
Because they aren't technically MsGraph custom action. I am thinking about keeping this with the custom actions for the calendar skill.